app/models/sample.rb

```diff
@@ -144,7 +144,6 @@ def self.search(search, eligible_pr_ids = [])
     if search
       search = search.strip
       results = where('samples.name LIKE :search
-        OR samples.sample_tissue LIKE :search
         OR samples.sample_location LIKE :search
```

Comment:
did your other PR take care of sample_location too?

Reply:
I still need to add the search on metadata... but that is valid for both fields.
I should remove location here too.